### DISCUSSION TITLE
Можно ли создать pterodactyl panel на github для создание вэб хостинга?

### DISCUSSION THREAD

**Discussion by Vik362352** [community/community#147416] [Discussions]
### Select Topic Area

Question

### Body

Можно ли создать pterodactyl panel на github.com для создания Web hosting, типо для nginx серверов и панели управления, просто лично мне не удобно работать с консолью...

**Comment by mecodeatlas** [comment]
Hi @Vik362352, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
